src/pages/billing/Project.js

```diff
@@ -296,7 +292,7 @@ const ProjectDetail = ({ authorizeAndLoadAccounts, billingAccounts, billingProje
         gridRowStart: 2
       }
     }, [
-      div({ style: { flex: 'none', padding: '0.625rem 1.25rem' } }, [
+      div({ style: { flex: 'none', padding: '0.625rem 1.25rem' }, 'aria-live': totalCost !== null ? 'polite' : 'off', 'aria-atomic': true }, [
```

Comment:
what about:
```suggestion
      div({ style: { flex: 'none', padding: '0.625rem 1.25rem' }, 'aria-live': !!totalCost ? 'polite' : 'off', 'aria-atomic': true }, [
```

Reply:
Looking at the API response, it seems like the value is a string though? The only falsy string would be `""`, I'd expect a cost of 0 to return `"0.00"`